crates/aranya-client/src/client.rs

```diff
@@ -120,7 +121,7 @@ impl Client {
     }
 
     /// Create a new graph/team with the current device as the owner.
-    pub async fn create_team(&mut self, cfg: TeamConfig) -> Result<TeamId> {
+    pub async fn create_team(&mut self, cfg: TeamConfig) -> Result<(TeamId, Vec<u8>)> {
```

Comment:
Type alias can be used here:
```suggestion
    pub async fn create_team(&mut self, cfg: TeamConfig) -> Result<(TeamId, InitCmd)> {
```